src-cpp/KafkaConsumerImpl.cpp

```diff
@@ -141,6 +141,14 @@ RdKafka::KafkaConsumerImpl::assignment (std::vector<RdKafka::TopicPartition*> &p
 }
 
 
+
+bool
+RdKafka::KafkaConsumerImpl::assignment_lost () {
+  return rd_kafka_assignment_lost(rk_);
```

Comment:
This needs an ternary check to avoid warnings on msvc:
` ..lost(rk_) ? true : false`

Reply:
this is so much easier in languages that give you squiggly red lines in your ide and are consistent across platforms.